css/style.css

```diff
@@ -0,0 +1,589 @@
+}
+
+nav > div.w-70 {
+  height: inherit;
+  width: 70%;
+}
+
+.form-inline .form-control {
+  width: 64px;
+}
+
+.email-subscription-top {
+  display: none;
+}
+
+.email-subscription-secret {
+  width: 70%;
+  height: auto;
+  background-color: #f4f5f6;
+}
+
+/* Main Section Starts here */
+
+main {
+  margin-top: 60px;
+  height: auto;
+  width: 100%;
+}
+
+article.subscribe-art {
+  height: 850px;
+}
+
+div.mag-thumb.current,
+#newsletter-img-top {
+  z-index: 6;
+  top: -210px;
+  left: 90px;
+  position: relative;
+}
+
+div.sub-left {
+  height: 280px;
+}
+
+div.last-section {
+  height: 800px;
+  margin: 0;
+  border: 2px solid green;
+}
+
+div.fn.image {
+  height: 280px;
+}
+
+div.f12.info {
+  position: absolute;
+  z-index: 7;
+  top: 650px;
+}
+
+article.newsletter {
+  position: relative;
+  z-index: 7;
+  top: 20px;
+}
+
+div.subscribe-sect {
+  margin-top: 100px;
+}
+
+article {
+  height: auto;
+}
+
+.caption {
+  position: relative;
+  top: -35px;
+  left: 0;
+  background-color: #fff;
+  width: 80px;
+  height: 35px;
+  opacity: 0.7;
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 24px;
+}
+
+.image-thumbnail {
+  width: 63px;
+}
+
+#versus-logo {
+  width: 30px;
+}
+
+.email-subscription,
+.email-subscription-top {
+  background-color: #f4f5f6;
+}
+
+#versus-container::before {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+#versus-container::after {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+.horizontal-line {
+  z-index: -1;
+  top: 15px;
+  height: 3px;
+}
+
+.svg-image {
+  z-index: 50;
+  top: 60px;
+}
+
+.right-news-cont {
+  display: none;
+}
+
+.section-logo {
+  width: 29px;
+}
+
+#newsletter-img-top {
+  width: 177px;
+}
+
+#footer-content {
+  flex-direction: column;
+}
+
+#footer-logo {
+  width: 251px;
+}
+
+.fa {
+  color: #656665;
+}
+
+#nl-img-cont {
+  height: 180px;
+}
+
+#newsletter-large-pic {
+  display: none;
+}
+
+div.smalldiv {
+  border: none !important;
+}
+
+@media screen and (max-width: 768px) and (min-width: 426px) {
+  .navbar-toggler-icon {
+    display: inline;
+  }
+
+  .feature-section,
+  .story-section,
+  .debate-section {
+    height: auto;
+  }
+
+  .feature-section,
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 4rem;
+  }
+
+  .feature-cont {
+    align-items: flex-start;
+  }
+
+  .image-thumbnail1 {
+    margin-left: auto;
+  }
+
+  .last-section {
+    flex-direction: row;
+  }
+
+  .email-subscription-top {
+    width: 40%;
+    height: auto;
+    display: inline;
+  }
+
+  .email-subscription {
+    width: 100%;
+    height: auto;
+  }
+
+  .right-news-cont {
+    display: none;
+  }
+
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 1rem;
+    align-self: flex-end;
+  }
+
+  #newsletter-large-pic {
+    display: inline;
```

Comment:
- [ ] There is a bootstrap class for this behavior, please use bootstrap whenever it is possible.